webapp/app/templates/fields/multiple_input_field.html

```diff
@@ -11,9 +11,14 @@
                     data.getAsString(str => {
                         let start_idx = 0
                         sub_fields.forEach(sub_field => {
-                            masked_str = $(sub_field).masked(str);
+                            try{
+                                masked_str = $(sub_field).masked(str);
+                            }catch (TypeError) {
+                                masked_str = str;
+                            }
```

Comment:
I've added this because it won't work on multiple Input fields without a `data-mask` attribute otherwise.